### PR TITLE
`[EXILED::API]` Adding ScaleNetworkIdentityObject

### PR DESCRIPTION
**Pull request by NotZer0Two**
## Description
**Describe the changes** 
Let's you scale down or up everything that has an NetworkIdentity

SPOILER_immagine
immagine

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?** (if this is a feature change)

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:

<br />

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [x] I have checked the project can be compiled
- [x] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [ ] I have checked no IL patching errors in the console

### Other
- [ ] Still requires more testing

### CODE REVIEW

**Comment by VALERA771** [comment]
I think you can use NetworkServer.Unspawn and NetworkServer.Spawn for cases when you need to move object for all players

File: EXILED/Exiled.API/Extensions/MirrorExtensions.cs [hunk 1]
```diff
@@ -342,6 +342,44 @@ public static void MessageTranslated(this Player player, string words, string tr
             }
         }
 
+        /// <summary>
+        /// Scale the object for a player.
+        /// </summary>
+        /// <param name="player">Target to send.</param>
+        /// <param name="identity">The <see cref="Mirror.NetworkIdentity"/> to move.</param>
+        /// <param name="scale">The scale to change.</param>
+        public static void MoveNetworkIdentityObject(this Player player, NetworkIdentity identity, Vector3 scale)
```
Comment (Misfiy) [inline EXILED/Exiled.API/Extensions/MirrorExtensions.cs:351]: You're naming the arg scale, but it does position

**Review by Misfiy — APPROVED** [review]

File: EXILED/Exiled.API/Extensions/MirrorExtensions.cs (hunk 1 above)
Comment (Misfiy) [inline EXILED/Exiled.API/Extensions/MirrorExtensions.cs:351]: Rename arg from scale

File: EXILED/Exiled.API/Extensions/MirrorExtensions.cs [hunk 2]
```diff
@@ -342,6 +342,44 @@ public static void MessageTranslated(this Player player, string words, string tr
...
+            player.Connection.Send(objectDestroyMessage, 0);
+            SendSpawnMessageMethodInfo?.Invoke(null, new object[] { identity, player.Connection });
+        }
+
+        /// <summary>
+        /// Scales the object for all the players.
+        /// </summary>
+        /// <param name="identity">The <see cref="NetworkIdentity"/> to move.</param>
+        /// <param name="scale">The scale to change.</param>
+        public static void MoveNetworkIdentityObject(this NetworkIdentity identity, Vector3 scale)
```
Comment (Misfiy) [inline EXILED/Exiled.API/Extensions/MirrorExtensions.cs:368]: Rename arg from scale

**Review by Misfiy — CHANGES_REQUESTED** [review]

**Comment by NotZer0Two** [comment]
> I think you can use NetworkServer.Unspawn and NetworkServer.Spawn for cases when you need to move object for all players. And isn't it a duplicate from your previous pr? (code is similar as I can see)

yes it is, in future i will probably do one method for everything but currently i want to them seperated when the update will be released i will do it

**Comment by VALERA771** [comment]
And is there any difference between this pr and #48?

**Comment by NotZer0Two** [comment]
> And is there any difference between this pr and #48?

Yes localscale its the only differents

**Comment by VALERA771** [comment]
There is local position. Seems that you haven't edited it

**Comment by NotZer0Two** [comment]
Im stupid and forgot to change the name

**Review by louis1706 — APPROVED** [review]

**Comment by VALERA771** [comment]
I'm about that you're still setting `localPosition` in method

File: EXILED/Exiled.API/Extensions/MirrorExtensions.cs [hunk 3]
```diff
@@ -342,6 +342,44 @@ public static void MessageTranslated(this Player player, string words, string tr
...
 
+        /// <summary>
+        /// Scale the object for a player.
+        /// </summary>
+        /// <param name="player">Target to send.</param>
+        /// <param name="identity">The <see cref="Mirror.NetworkIdentity"/> to scale.</param>
+        /// <param name="scale">The scale the object needs to be.</param>
+        public static void ScaleNetworkIdentityObject(this Player player, NetworkIdentity identity, Vector3 scale)
+        {
+            identity.gameObject.transform.localPosition = scale;
```
Comment (VALERA771) [inline EXILED/Exiled.API/Extensions/MirrorExtensions.cs:353]: Should be scale

File: EXILED/Exiled.API/Extensions/MirrorExtensions.cs [hunk 4]
```diff
@@ -342,6 +342,44 @@ public static void MessageTranslated(this Player player, string words, string tr
...
+        }
+
+        /// <summary>
+        /// Scales the object for all the players.
+        /// </summary>
+        /// <param name="identity">The <see cref="Mirror.NetworkIdentity"/> to scale.</param>
+        /// <param name="scale">The scale the object needs to be.</param>
+        public static void ScaleNetworkIdentityObject(this NetworkIdentity identity, Vector3 scale)
+        {
+            identity.gameObject.transform.localPosition = scale;
```
Comment (VALERA771) [inline EXILED/Exiled.API/Extensions/MirrorExtensions.cs:370]: Same as above

**Review by VALERA771 — CHANGES_REQUESTED** [review]

**Comment by Misaka-ZeroTwo** [comment]
pls fix build errors

**Review by Misfiy — APPROVED** [review]

**Review by VALERA771 — APPROVED** [review]

**Pull request merged**